plugins/plugin-chart-echarts/src/Timeseries/Area/controlPanel.tsx

```diff
@@ -251,6 +252,18 @@ const config: ControlPanelConfig = {
             },
           },
         ],
+        [
+          {
+            name: 'show_value',
+            config: {
+              type: 'CheckboxControl',
+              label: t('Show Value'),
+              default: showValue,
+              renderTrigger: true,
+              description: t('Show the value on top of the chart'),
```

Comment:
Could we rephrase this to something like "Show series values on the chart" (on top of the chart might make it sound like it's on top of the whole chart rectangle).

Reply:
You are right!